site/docs/tasks/custom-taxonomy.md

```diff
@@ -51,3 +51,123 @@ helm install fybrik fybrik-charts/fybrik -n fybrik-system --wait --set-file taxo
 ```
 The `--set-file` flag will pass in your custom `taxonomy.json` file to use for taxonomy validation in Fybrik.
 If this flag is not provided, Fybrik will use the default `taxonomy.json` file with no layers compiled into it. 
+
+
+After installing the fybrik helm chart, it is possible to upgrade Fybrik with an updated custom taxonomy file (`taxonomy.json`) with the following command:
+
+```bash
+helm upgrade fybrik fybrik-charts/fybrik -n fybrik-system --wait --set-file taxonomyOverride=taxonomy.json
+```
+
+
+## Examples of changing taxonomy
+
+- Example 1: Add new intent for FybrikApplication
+
+Starting with installing Fybrik using the [`quickstart guide`](https://fybrik.io/latest/get-started/quickstart/) but stop before the command of `helm install fybrik fybrik-charts/fybrik -n fybrik-system --wait`
```

Comment:
Minor change suggested:

Follow the [`quickstart guide`](https://fybrik.io/latest/get-started/quickstart/)  but stop before the command `helm install fybrik fybrik-charts/fybrik -n fybrik-system --wait`...